app/assets/javascripts/admin/addon/components/theme-settings-editor.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="settings-editor settings">
```

Comment:
I don't see the `setting` class being used for styling in `customize.scss`

```suggestion
<div class="settings-editor">
```

Reply:
done